src/document-converter.ts

```diff
@@ -219,10 +219,14 @@ export class DocumentConverter extends DocumentProcessor {
     const filteredImports = [];
     for (const import_ of DocumentConverter.getAllHtmlImports(this.document)) {
       if (!isImportWithDocument(import_)) {
-        console.warn(
-            `${this.originalPackageName} ${this.originalUrl}: The document ` +
-            `referenced using URL '${import_.originalUrl}' could not be ` +
-            `loaded and was ignored.`);
+        console.warn(new Warning({
+          code: 'import-ignored',
+          message: `The import referenced by URL '${import_.originalUrl}' ` +
```

Comment:
No need to include info that's present in the source code, it'll be logged with a squiggle under the original source code. So these messages could just be:

```
message: `Import could not be loaded and will be ignored.`,
```

Reply:
Ok, I'll update these messages and merge.